<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultFrm</class>
 <widget class="QWidget" name="ResultFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version 1 Only</string>
   </property>
  </widget>
  <widget class="QListWidget" name="v1List">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="v2List">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>171</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version 2 Only</string>
   </property>
  </widget>
  <widget class="QListWidget" name="difList">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>171</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Different files</string>
   </property>
  </widget>
  <widget class="QListWidget" name="failedList">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>171</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Failed files</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version 1 Path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="v1Edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="v2Edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version 2 Path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cpButton1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制文本到剪贴板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cpButton2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制文本到剪贴板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cpButton3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制文本到剪贴板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cpButton4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制文本到剪贴板</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
